features/google/src/components/activitiesOverTimeStory/activitiesOverTimeStory.css

```diff
@@ -4,11 +4,11 @@
 }
 
 .datepicker .space-right {
-    margin-right: 10px;
+    margin-right: calc(var(--gdu) * 2);
```

Comment:
Gone for the low end here? You could have picked `* 3` too.

Reply:
In this kind of cases, I checked how it looked in the browser and picked the option which looked more like Figma. 

However... I couldn't find the Figma file for this particular case, so I just picked `* 2` and checked that it looked good in different browsers. 🙈